backend/config.go

```diff
@@ -103,13 +103,15 @@ type Proxy struct {
 
 func (c *GrafanaCfg) proxy() Proxy {
 	if v, exists := c.config[proxy.PluginSecureSocksProxyEnabled]; exists && v == strconv.FormatBool(true) {
+		allowInsecure, _ := strconv.ParseBool(c.Get(proxy.PluginSecureSocksProxyAllowInsecure))
```

Comment:
Should check for error/unable to parse here.

Reply:
Done